tests_new/integration_tests/core/environment/global_conftest.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import re
+
+import pytest
+
+from integration_tests.client import GqlError
+from integration_tests.core.environment.queries import create_environment, get_environment, delete_environment
+from integration_tests.utils import poller
+
+log = logging.getLogger(__name__)
+
+
+@poller(check_success=lambda env: not re.match(r'.*IN_PROGRESS|PENDING', env.stack.status, re.IGNORECASE))
+def check_env_ready(client, env_uri):
+    env = get_environment(client, env_uri)
+    log.info(f'polling {env_uri=}, new {env.stack.status=}')
+    return env
+
+
+def create_env(client, group, org_uri, account_id, region):
+    new_env_uri = create_environment(
+        client, name='testEnvA', group=group, organizationUri=org_uri, awsAccountId=account_id, region=region
+    )['environmentUri']
+    return check_env_ready(client, new_env_uri)
+
+
```

Comment:
As next steps, maybe we could add a check_env_deleted. Something that ensures that everything is cleaned-up before the second run of the tests

Reply:
I think this should be a test on each own. Delete environment returns true/false so we need to assert that when it returns true the environment is gone.